src/component/App/App.js

```diff
@@ -0,0 +1,14 @@
+import './App.css';
+import ButtonPanel from '../buttonPanel';
+import Display from '../Display';
+
+function App() {
+  return (
+    <div className="App">
+      <ButtonPanel />
+      <Display />
+    </div>
+  );
+}
+
+export default App;
```

Comment:
- [ ] I noticed a mixture of your components and styling files within the component older. This makes your codebase difficult to read, I highly recommend you create a separate folder that stores your stylesheets and you can name them according to the component they belong to, your component folder should just consist of components as this will enhance the professional standing out of your repository.

Reply:
I'm not sure this is the case because in react docs of how to add CSS module they specify that you need to add it within react component, not in file contain all of the CSS